app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+  after_save :update_posts_counter
+
+  def recent_five_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  private
+
+  def update_posts_counter
+    user.increment!(:posts_counter)
```

Comment:
- [ ] this should be author.increment!()

Reply:
Dear @VICTOR-CHUKWU 
However much it could be a good suggestion to apply, I don't think its a mandatory assignment to apply, since it is just a naming. Having author and user literally means the same thing and for best naming practice, "user" would be the best option rather than changing it to author. Kindly let me know what could be the reason behind changing the name as you require. Thank you.

Reply:
please know that it is part of project requirement, you need to reference user as author :+1: